### PR TITLE
[WIP] Create a packer template for the devel box

### PR DESCRIPTION
**Pull request by johnpmitsch**
This commit adds a packer config for creating an image of the devel box.
You can use this by running:

```
cd templates
packer build katello-devel.json
```

### CODE REVIEW

File: templates/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This directory contains [packer](https://www.packer.io/) templates to create images from provisioned VMs
```
Comment (ehelms) [inline templates/README.md]: How we name this folder `packer` for clarity?

**Comment by ehelms** [comment]
@johnpmitsch What's left to complete this? Are you still planning to use it?

**Comment by johnpmitsch** [comment]
@ehelms I was able to get a box created and running using the installer PR workaround above, but there was some trouble getting the dev server started. It takes some effort to debug anything with packer boxes so I'm still figuring things out.

I still would like to pursue this, if it works best I can close and then re-open the PR when it is ready since there is no review needed right now

**Comment by johnpmitsch** [comment]
Going to close this out and reopen when i have more time to investigate

**Pull request closed without merging**